src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -214,16 +214,16 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
     private String getSubmissionFormTableHeaderFragmentHtml(String questionNumberString, String responseNumberString) {
         StringBuilder tableHeaderFragmentHtml = new StringBuilder();
         String tableHeaderFragmentTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_HEADER_FRAGMENT;
-
-        for(int i = 0 ; i < numOfRubricChoices ; i++) {
+        
+        for(int i = 0; i < numOfRubricChoices; i++) {
             String tableHeaderCell = 
                     FeedbackQuestionFormTemplates.populateTemplate(tableHeaderFragmentTemplate,
                             "${qnIndex}", questionNumberString,
                             "${respIndex}", responseNumberString,
                             "${col}", Integer.toString(i),
-                            "${rubricChoiceValue}", rubricChoices.get(i));
+                            "${rubricChoiceValue}", (rubricChoices.get(i) + " (" + (numOfRubricChoices-i) + ")"));
             tableHeaderFragmentHtml.append(tableHeaderCell + Const.EOL);
-        }
+         }
```

Comment:
Still a few coding style errors here:
- `for (...) {`: notice the whitespace before/after the brackets
- `(numOfRubricChoices - i)`: ditto, before/after the minus sign; similar for other operators like + * /
- `}` was in the correct position before, revert it?